pkgs/applications/gis/qgis/unwrapped.nix

```diff
@@ -132,31 +138,76 @@ in mkDerivation rec {
 
   cmakeFlags = [
     "-DWITH_3D=True"
-    "-DWITH_PDAL=TRUE"
   ] ++ lib.optional (!withWebKit) "-DWITH_QTWEBKIT=OFF"
+    ++ lib.optional stdenv.isLinux "-DWITH_PDAL=TRUE"
```

Comment:
Why this change ? it seems to build fine on Mac with this.

Reply:
Good call -- it was broken a long time ago and I just assumed that was still true. Fixed since October with https://github.com/NixOS/nixpkgs/pull/195216. Will revise.